MailCore/Cache/AccountManager/AccountManager.swift

```diff
@@ -33,12 +33,12 @@ public extension InfomaniakNetworkLoginable {
     func apiToken(using code: String, codeVerifier: String) async throws -> ApiToken {
         try await withCheckedThrowingContinuation { continuation in
             getApiTokenUsing(code: code, codeVerifier: codeVerifier) { result in
-//                switch result {
-//                case .success(let token):
-//                    continuation.resume(returning: token)
-//                case .failure(let error):
-//                    continuation.resume(throwing: error)
-//                }
+                switch result {
```

Comment:
We should now be able to use the async await version of this function provided by login